protocol/skeleton.go

```diff
@@ -14,7 +15,7 @@ func getHeaders(clt proto.V1Client, req *proto.GetHeadersRequest) ([]*types.Head
 		return nil, err
 	}
 
-	headers := []*types.Header{}
+	headers := make([]*types.Header, 0)
```

Comment:
If you know the size, please reserve it

Reply:
Reserved 👍 

[5d0f6c7](https://github.com/0xPolygon/polygon-edge/pull/548/commits/5d0f6c7f6d66a7d1a0e779144d350b0a0d464eec)